socorro/unittest/external/es/base.py

```diff
@@ -123,14 +117,11 @@
         'is_returned': True,
         'is_mandatory': False,
         'storage_mapping': {
-            'type': 'multi_field',
+            'type': 'string',
             'fields': {
-                'os_name': {
-                    'type': 'string'
-                },
```

Comment:
Yes, this is a mapping change that happened in 1.x and has been removed from 2.x forward. This was also some leftover from my work on 5.x, but I believe it is a fine thing to keep. 

So, the new mapping format for multi-fields is this: define the default field in the root, and then add a ``fields`` key in which you define other sub-fields. The two syntaxes are strictly equivalent, but the new one is future-proof.